thetis/turbulence.py

```diff
@@ -682,7 +700,9 @@ def postprocess(self):
             set_func_min_val(self.viscosity, o.visc_min)
             set_func_min_val(self.diffusivity, o.diff_min)
 
+            # print_output(' -----')
```

Comment:
I presume this is some frequently used debugging output? Can this go into a `if self._debug_output:` block?

Reply:
Yes. Now moved to `print_debug` function.